typescript/agentkit/package.json

```diff
@@ -18,7 +18,7 @@
     "format:check": "prettier -c .prettierrc --check \"**/*.{ts,js,cjs,json,md}\"",
     "check": "tsc --noEmit",
     "test": "npx jest --no-cache --testMatch='**/*.test.ts'",
-    "test:dry-run": "npm install && npm ci && npm publish --dry-run",
+    "test:dry-run": "pnpm install --frozen-lockfile && pnpm publish --dry-run",
```

Comment:
This is a weird command, I haven't noticed it before. Doesn't seem it's used, I think we should just remove it.